TLM/TLM/UI/MainMenu/MainMenuWindow.cs

```diff
@@ -15,10 +15,9 @@ namespace TrafficManager.UI.MainMenu {
     using UnityEngine;
     using TrafficManager.Lifecycle;
 
-    public class MainMenuWindow
+    public partial class MainMenuWindow
```

Comment:
I'm not a fan of partial classes since they are mostly for externally generated code (using some kind of UI designer or something)

Reply:
UI code grows long very quick, there isn't an UI designer file to hide the layout and setup code, so i am trying to not have it in the same file where the rest of the logic sits.